contracts/enforcer/HolographRoyalties.sol

```diff
@@ -696,4 +749,31 @@ contract HolographRoyalties is Admin, Owner, Initializable {
   function getTokenAddress(string memory tokenName) public view returns (address) {
     return _getTokenAddress(tokenName);
   }
+
+  /**
+   * @notice Used to wrap function calls to check if they return without revert regardless of return type.
+   * @dev
```

Comment:
@ACC01ADE Would you be able to add an extra note here on the logic. I know you mentioned it while we were chatting yesterday, but wasn't sure the best way to articulate it in writing

Reply:
Replied in the `src` version of the smart contract. [Here](https://github.com/holographxyz/holograph-protocol/pull/93#discussion_r1038542139)